### PR TITLE
 - DRILL-5507 Made verbose info logging message debug level and print…

### PR DESCRIPTION
**Pull request by ilooner-mapr**
…ed it less frequently

### CODE REVIEW

**Comment by ilooner-mapr** [comment]
@paul-rogers

File: exec/java-exec/src/main/java/org/apache/drill/exec/store/schedule/BlockMapBuilder.java [hunk 1]
```diff
@@ -228,6 +230,7 @@ public EndpointByteMap getEndpointByteMap(FileWork work) throws IOException {
 
     // Find submap of ranges that intersect with the rowGroup
     ImmutableRangeMap<Long,BlockLocation> subRangeMap = blockMap.subRangeMap(rowGroupRange);
+    Set<String> noDrillbitHosts = Sets.newHashSet();
```
Comment (vrozov) [inline exec/java-exec/src/main/java/org/apache/drill/exec/store/schedule/BlockMapBuilder.java]: Consider `final Set<String> noDrillbitHosts = logger.isDebugEnabled() ? Sets.newHashSet() : null;`

File: exec/java-exec/src/main/java/org/apache/drill/exec/store/schedule/BlockMapBuilder.java [hunk 2]
```diff
@@ -246,12 +249,16 @@ public EndpointByteMap getEndpointByteMap(FileWork work) throws IOException {
         DrillbitEndpoint endpoint = getDrillBitEndpoint(host);
         if (endpoint != null) {
           endpointByteMap.add(endpoint, bytes);
-        } else {
-          logger.info("Failure finding Drillbit running on host {}.  Skipping affinity to that host.", host);
+        } else if (logger.isDebugEnabled()) {
```
Comment (vrozov) [inline exec/java-exec/src/main/java/org/apache/drill/exec/store/schedule/BlockMapBuilder.java]: and `} else if (noDrillbitHosts != null && noDrillbitHosts.add(host)) { logger.debug(...); }`

**Comment by ilooner-mapr** [comment]
Applied comments

**Pull request closed without merging**